### PR TITLE
Wired up Persistence to Server

### PR DESCRIPTION
**Pull request by rposborne**

### CODE REVIEW

File: src/sync/runSync.js [hunk 1]
```diff
@@ -23,7 +36,19 @@ function postObservations (observations, sentAt) {
       sent_at: sentAt,
       observations
     }
-    console.log(dataToPost)
-    return resolve()
+    console.log('data to post', dataToPost)
+
+    return fetch(
+      new AuthenticatedRequest('/api/observation_sessions', {
+        method: 'POST',
```
Comment (kwokster10) [inline src/sync/runSync.js]: Do we need headers here?

File: src/components/Conclusion.vue [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="bg-washed-blue">
```
Comment (kwokster10) [inline src/components/Conclusion.vue]: The `bg-washed-blue` class is at the `App` level. Can we update this to `conclusion`?

File: src/store/observations.js [hunk 3]
```diff
@@ -4,54 +4,50 @@ import Vue from 'vue'
...
-      },
-      789: {
-        id: 123,
-        locationId: 123,
-        subject: 'Pamela',
-        actionName: 'walking',
-        modifierName: 'quickly'
-      }
-    }
+    data: {}
```
Comment (kwokster10) [inline src/store/observations.js]: The `Observations` view depends on this for rendering so I think this needs to be null as default.

File: src/sync/runSync.js [hunk 4]
```diff
@@ -23,7 +36,19 @@ function postObservations (observations, sentAt) {
       sent_at: sentAt,
       observations
     }
-    console.log(dataToPost)
-    return resolve()
+    console.log('data to post', dataToPost)
```
Comment (kwokster10) [inline src/sync/runSync.js]: remove?

**Review by kwokster10 — CHANGES_REQUESTED** [review]

**Review by bhaibel — APPROVED** [review]

**Review by kwokster10 — APPROVED** [review]

**Pull request merged**